dns_sd_windows.c

```diff
@@ -9,16 +9,16 @@
  * which is Licensed under Public Domain
  */
 
+#include "debug.h"
+#include "dns_sd.h"
+#include "iio-private.h"
+#include "mdns.h"
+
```

Comment:
In the other files I moved the includes around so that they are in alphabetical order. I guess I can move these in a separate commit.

Including local headers first allows you to make sure that the local headers include their dependencies, so this should be the preferred way.

Reply:
Yes, I noticed you ordering them in alphabetical order which is nice...